pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -130,6 +125,13 @@ func (r *ClusterInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}
 
+	// Get distribution info
+	newStatus.DistributionInfo, err = r.getDistributionInfo(ctx, product)
```

Comment:
Line 123 will be dead code. https://github.com/stolostron/multicloud-operators-foundation/pull/526/files#diff-ac6674a4f427b422665e2fbea90ad4e776f2946ae22cc9974e4eb9045d5a4312R123